### DISCUSSION TITLE
3.1.0rc4

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#26436] [General]
Hello Superset Community,

Superset 3.1.0 RC4 is up for testing. Check the [Testing a Release Candidate (RC)](https://github.com/apache/superset/wiki/Release-Process#testing-a-release-candidate-rc) section of our [Release Process](https://github.com/apache/superset/wiki/Release-Process) for information on how to test and report issues with the release candidate. 

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/3.1.0rc4/

Git tag for the release:
https://github.com/apache/superset/tree/3.1.0rc4

The Change Log for the release:
https://github.com/apache/superset/blob/3.1.0rc4/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/3.1.0rc4/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/3.1.0rc4'>3.1.0rc4</a>.</em>

**Comment by mateen-digi** [comment]
In the new Bubble chart( E-chart) cross filtering is not working , is that expected
http://34.217.96.197:8080/superset/dashboard/12/?native_filters_key=buSLZ31v26lAKKUSYNH1dEBHaUgfVcMH3ak6dv6goGNzrSSZfiUeoCbSuMsoqCoY

**Reply by michael-s-molina** [reply]
Hi @mateen-digi. Yes, it's expected. Cross-filtering is not implemented for this chart type yet.

**Reply by mateen-digi** [reply]
Thanks @michael-s-molina

**Comment by ketulgupta1995** [comment]
n 2.x superset there is an option in sql lab page to add jinja parameters for the query we are executing as json, is this expected to be not present in the 3.1?

<img width="236" alt="image" src="https://github.com/apache/superset/assets/11848636/b1dd0f69-73b9-49a6-bae7-57d22c363827">

**Reply by michael-s-molina** [reply]
You need to set `ENABLE_TEMPLATE_PROCESSING` feature flag to `True` to see that option.

**Comment by ketulgupta1995** [comment]
I missed that 👍

**Comment by iamsaso** [comment]
I was testing this release and saw that Playwright doesn't take full_page screenshots, after digging through the code, I see that we are not setting `full_page=True` here https://github.com/apache/superset/blob/master/superset/utils/webdriver.py#L234 - Is this something we should enable?

**Reply by michael-s-molina** [reply]
Hi @iamsaso. Thanks for testing the RC. That's a good question and the best person to answer it would be @kgabryje. Do you mind opening an issue for this and tagging him? This is definitely not a release blocker but something we can cherry-pick in a patch release.

**Reply by iamsaso** [reply]
got my answer here, ty! https://github.com/apache/superset/issues/24948#issuecomment-1888992027

**Comment by medhinishetty** [comment]
Hi,
I am getting the SQLLAB Error while running Time Functions in Superset metadatabase while using Superset 3.1.0 RC4 
Here is the issue created with complete description :https://github.com/apache/superset/issues/26643

**Reply by mdeshmu** [reply]
@betodealmeida can you please have a look.

**Reply by michael-s-molina** [reply]
Thanks @medhinishetty for creating the issue. As soon as a fix is provided, we'll cherry-pick it into the next patch.

**Comment by michael-s-molina** [comment]
Closing the discussion given that 3.1.0 was released! Any unresolved issues will be fixed in subsequent patches. Many thanks to everyone who contributed and participated in this release!
